generators/server/index.js

```diff
@@ -706,16 +716,6 @@ module.exports = JhipsterServerGenerator.extend({
             }
         },
 
-        writeServeri18nFiles: function () {
-
-            // install all files related to i18n if translation is enabled
-            if (this.enableTranslation) {
-                this.installI18nResFilesByLanguage(this, SERVER_MAIN_RES_DIR, 'en');
-                this.installI18nResFilesByLanguage(this, SERVER_MAIN_RES_DIR, 'fr');
-            }
-            this.template(SERVER_MAIN_RES_DIR + 'i18n/_messages_en.properties', SERVER_MAIN_RES_DIR + 'i18n/messages.properties', this, {});
```

Comment:
but if the server sub gene is run directly, which is also feasible, this file will not be generated and the app will fail when run. since the `messages.properties` file is required by default for spring I would rather have that done here and handle only the `en` and `fr` files in language sub gen.

may be the default `messages.properties` template should be placed under the server sub gen resource folder